govc/library/rm.go

```diff
@@ -28,6 +28,8 @@ import (
 	"github.com/vmware/govmomi/vapi/rest"
 )
 
+var fmtMultiMatch = "%q matches %d items, library id must be specified"
```

Comment:
Hi @dougm,

This should be a constant.

Reply:
Hi @dougm,

Also, maybe consider the following - ([go playground link](https://play.golang.org/p/qt3Uc-No6c2))

```golang
package main

import (
	"fmt"
	"log"
)

// ErrMultiMatch is an error returned when a query returns more than one result.
type ErrMultiMatch struct {
	// Type is the type of object being queried.
	Type string

	// TokenKey is the key used to perform the query.
	TokenKey string

	// TokenVal is the value used to perform the query.
	TokenVal string

	// Count is the number of objects returned.
	Count int
}

// Error returns the error string.
func (e ErrMultiMatch) Error() string {
	return e.String()
}

// String returns the same value as Error().
func (e ErrMultiMatch) String() string {
	return fmt.Sprintf("%q=%q matches %d items, %q id must be specified", e.TokenKey, e.TokenVal, e.Count, e.Type)
}

func main() {
	err := ErrMultiMatch{
		Type:     "library",
		TokenKey: "name",
		TokenVal: "MyContentLibrary",
		Count:    3,
	}
	log.Fatal(err)
}
```

Prints the following:

```shell
2009/11/10 23:00:00 "name"="MyContentLibrary" matches 3 items, "library" id must be specified
```

Reply:
Thanks @akutz, I like your error type and have integrated into this PR.